### ISSUE TITLE
 instead of your function 'torch.linspace() 

### ISSUE THREAD

**Issue by longmalongma** [qway/nerfmeshes#2]
Can I use the function 'inputs_normal = inputs/tf.reduce_max(tf.abs(inputs))' to instead of your'torch.linspace() '? Because they can all normalize tensor to [-1,1].
